src/lib.rs

```diff
@@ -153,7 +190,7 @@ where
     /// This method does not retrieve any value that goes with the
     /// option. If the option requires an value, such as in
     /// `--option=value`, then you should call
-    /// [`value`](#method.value) after getting the option.
+    /// [`value`] or [`value`] after getting the option.
```

Comment:
Remove duplicate

Reply:
Oops, it used to say "value or value_str" but then value_str got removed